partials/tabs/about.html

```diff
@@ -3,8 +3,16 @@
         <div class="col-md-8 col-md-offset-2">
             <div class="title-get-involved">about</div>
             <br>
-            <div class="get-description">
-                
+            <div class="about-description">
```

Comment:
with ```p``` the font size is coming a bit small, but otherwise it is quite alright
![screencapture-localhost-8090-1491758306301](https://cloud.githubusercontent.com/assets/13018570/24839379/d5ccb9c2-1d76-11e7-961e-41aeaead4d16.png)